frontend/src/components/pages/admin/AdminRouting.tsx

```diff
@@ -36,12 +38,14 @@ const AdminRouting = (): React.ReactElement => {
                 exact
                 path={Routes.USER_DATABASE_PAGE}
                 roles={["Admin"]}
+                title="Users"
```

Comment:
i feel like this should be `Database` to keep consistent with the page name - thought database is kind of confusing in the first place lol